plugin-dev/Source/Sentry/Private/Desktop/SentrySubsystemDesktop.cpp

```diff
@@ -33,6 +33,12 @@ void SentrySubsystemDesktop::InitWithSettings(const USentrySettings* settings)
 #endif
 
 	const FString HandlerPath = FPaths::Combine(FSentryModule::Get().GetBinariesPath(), HandlerExecutableName);
+	
+	if (!FPaths::FileExists(HandlerPath))
+	{
+		UE_LOG(LogSentrySdk, Error, TEXT("Crashpad_handler.exe doesn't exist - Is it included in the UGS Binaries zip or compiled locally?."));
+		return;
```

Comment:
There is no mechanism allowing us to do that, so printing the error message is the only thing we can do atm

Reply:
@tustanivsky Yeah we assumed so, our concern was if it failed to init and returned out, would the editor crash later on because it tried to access Sentry code somewhere?

Reply:
handler executable isn't accessed directly in such a way so I don't think that could happen.